conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ConjureJerseyFeature.java

```diff
@@ -17,42 +17,43 @@
 package com.palantir.conjure.java.server.jersey;
 
 import com.fasterxml.jackson.jaxrs.cbor.JacksonCBORProvider;
+import com.google.errorprone.annotations.CheckReturnValue;
 import com.palantir.conjure.java.serialization.ObjectMappers;
+import com.palantir.logsafe.Preconditions;
 import com.palantir.tracing.jersey.TraceEnrichingFilter;
-import com.palantir.tritium.metrics.registry.SharedTaggedMetricRegistries;
 import javax.ws.rs.core.Feature;
 import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
 
 public enum ConjureJerseyFeature implements Feature {
-    INSTANCE;
-
-    private final JerseyServerMetrics metrics;
-
-    ConjureJerseyFeature() {
-        this.metrics = JerseyServerMetrics.of(SharedTaggedMetricRegistries.getSingleton());
-    }
 
-    public JerseyServerMetrics getMetrics() {
-        return metrics;
-    }
+    /**
+     * Prefer {@link #builder()} which allows passing in a real {@link ExceptionListener} which records internal errors.
+     */
+    INSTANCE;
```

Comment:
Thoughts on explicitly deprecating this? I didn't do it originally because I didn't want to flag the test-only usages, but looking at sourcegraph there are actually a few prod users... https://sourcegraph.p.b/search?q=ConjureJerseyFeature.INSTANCE+lang:java&patternType=literal

Reply:
I'm okay with a deprecation if you'd like to PR uses out of "large internal" :-)